### PR TITLE
Add academy index page

### PR DESCRIPTION
**Pull request by mnmkng**
Related academy PR https://github.com/apify/apify-web/pull/1802

### CODE REVIEW

File: content/academy/homepage_content.json
Comment (m-murasovs) [inline content/academy/homepage_content.json:11]:
```suggestion
            "description": "The Apify platform is the best place to run your scrapers and automations in the cloud. Learn what an actor is, how to turn your program into an actor, and how to deploy it.",
```

**Review by m-murasovs — APPROVED** [review]

File: content/academy/homepage_content.json
Comment (nguyeda1) [inline content/academy/homepage_content.json:5]: These links will not work on `developers.apify.com/academy` (the homepage) unless we change the links here to a format: `academy/[article-path]`. Or we hardcode the academy prefix to the `CardItem` on web. I would go with the first option. What do you think? @mnmkng @m-murasovs

Comment (mnmkng) [reply content/academy/homepage_content.json:5]: I think I had the `academy` there before, because intuitively it made sense like this, but it did not work locally. But you did some changes to the link property inside the other PR so this is a follow up to that?

Comment (nguyeda1) [reply content/academy/homepage_content.json:5]: Yes I updated the links logic in this commit. It should be working if once you update the links here to the proposed format

Comment (mnmkng) [reply content/academy/homepage_content.json]: done

**Review by nguyeda1 — APPROVED** [review]

**Comment by mnmkng** [comment]
Should we merge this before or after https://github.com/apify/apify-web/pull/1802 is merged?

**Comment by nguyeda1** [comment]
> Should we merge this before or after [apify/apify-web#1802](https://github.com/apify/apify-web/pull/1802) is merged?

For this instance **after**. You just added logic for displaying the academy homepage on the web and since it won't be linked from anywhere until you merge the changes here, it should do no harm. I think :D

**Pull request merged**
